Cargo.toml

```diff
@@ -1,2 +1,81 @@
+[workspace]
+resolver = "2"
+members = ["zk-sdk"]
+
+[workspace.package]
+authors = ["Anza Maintainers <maintainers@anza.xyz>"]
+repository = "https://github.com/solana-program/zk-elgamal-proof"
+homepage = "https://anza.xyz/"
+license = "Apache-2.0"
+edition = "2021"
+
+[workspace.lints.rust]
+warnings = "deny"
+
+[workspace.lints.rust.unexpected_cfgs]
+level = "warn"
+check-cfg = ['cfg(target_os, values("solana"))']
+
 [workspace.metadata.cli]
-solana = "2.1.0"
+solana = "2.2.11"
+
+# Specify Rust toolchains for rustfmt, clippy, and build.
+# Any unprovided toolchains default to stable.
+[workspace.metadata.toolchains]
+format = "nightly-2024-11-22"
+lint = "nightly-2024-11-22"
```

Comment:
Since you're using 1.87 for the stable toolchain, which I'm assuming was taken from Agave master, let's line these up too and use 2025-03-29

Reply:
Yep, done!